wellcomeml/viz/visualize_clusters.py

```diff
@@ -0,0 +1,31 @@
+from bokeh.plotting import figure, show
+from wellcomeml.viz.palettes import Wellcome33
+from bokeh.io import output_notebook
+output_notebook()
+
+
+def visualize_clusters(reduced_points: list):
```

Comment:
Let's make it customisable! Maybe you can add `radius` and `alpha` as parameters. I'd also include the following parameters

`radius: float, alpha: float, output_file_path: str, output_notebook: bool`

If someone says `output_notebook=True`, then you call output_notebook, if they pass a file_name, then you call `output_file`.

Reply:
I have added `radius: float, alpha: float` and `output_notebook: bool` that prints in the notebook if True, on a new page otherwise.
I still haven't added `output_file_path: str` because I don't understand what the string would be. The file path or the file name?

Thanks for the comment!

Reply:
Thanks for the additions. So at the moment you call `ouput_file('cluster_viz.html')`, that's hard coded. with output_file_path you would let the user specify it (maybe have a default). so:

`output_file_path: str = 'cluster_viz.html'`

Btw, can we make all variable lower cased? :) That's usually the convention we follow with python variables inside functions

Reply:
True, it's hard coded so it's better to change it. I also forgot to add in the comment a short explanation of the new parameters.

Regarding lower case it's fine, i just used upper case for not having overriding memory :)